src/graphics.c

```diff
@@ -1739,6 +1739,41 @@ const u16 gPokedexSearchMenu_Pal[] = INCBIN_U16("graphics/pokedex/search_menu.gb
 const u32 gPokedexSearchMenu_Gfx[] = INCBIN_U32("graphics/pokedex/search_menu.4bpp.lz");
 const u32 gPokedexSearchMenuNational_Tilemap[] = INCBIN_U32("graphics/pokedex/search_menu_national.bin.lz");
 const u32 gPokedexSearchMenuHoenn_Tilemap[] = INCBIN_U32("graphics/pokedex/search_menu_hoenn.bin.lz");
+// Pokedex HGSS
+#if POKEDEX_PLUS_HGSS == TRUE
+const u16 gPokedexPlusHGSS_Default_Pal[]             = INCBIN_U16("graphics/pokedex/hgss/HGSS_palette_default.gbapal");
```

Comment:
Alignment here is weird. Let's remove it to keep consistant with previous graphics.